server/broadcast.go

```diff
@@ -474,7 +474,7 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string,
 		return nil, nil
 	}
 
-	glog.Infof("Trying to transcode segment nonce=%d seqNo=%d", nonce, seg.SeqNo)
+	glog.Infof("Trying to transcode segment mid=%v nonce=%d seqNo=%d", cxn.mid, nonce, seg.SeqNo)
```

Comment:
From a glance, it looks like manifestID= is used in most places at least in the segment submission/receipt flows in [broadcast.go](https://github.com/livepeer/go-livepeer/blob/master/server/broadcast.go). For consistency, let's use manifestID= here as well instead of mid=